### PR TITLE
remove unused models-generation code (all, complete and default models)

### PR DESCRIPTION
**Pull request by OriginLabs-Iguernlala**

### CODE REVIEW

File: src/lib/frontend/frontend.ml [hunk 1]
```diff
@@ -225,7 +225,7 @@ module Make(SAT : Sat_solver_sig.S) : S with type sat_env = SAT.t = struct
     with
     | SAT.Sat t ->
       print_status (Sat (d,t)) (Steps.get_steps ());
-      if get_model () then SAT.print_model ~header:true (get_fmt_mdl ()) t;
+      (*if get_model () then SAT.print_model ~header:true (get_fmt_mdl ()) t;*)
```
Comment (ACoquereau) [inline src/lib/frontend/frontend.ml:228]:
```suggestion
```

File: src/lib/frontend/frontend.ml [hunk 2]
```diff
@@ -234,7 +234,7 @@ module Make(SAT : Sat_solver_sig.S) : S with type sat_env = SAT.t = struct
       env , false, dep
     | SAT.I_dont_know t ->
       print_status (Unknown (d, t)) (Steps.get_steps ());
-      if get_model () then SAT.print_model ~header:true (get_fmt_mdl ()) t;
+      (*if get_model () then SAT.print_model ~header:true (get_fmt_mdl ()) t;*)
```
Comment (ACoquereau) [inline src/lib/frontend/frontend.ml:237]:
```suggestion
```
This can be remove

**Comment by ACoquereau** [comment]
In expr.ml, is_in_model and associated functions or not useful anymore

**Comment by ACoquereau** [comment]
> In expr.ml, is_in_model and associated functions or not useful anymore

https://github.com/OriginLabs-Iguernlala/alt-ergo/pull/2

File: src/lib/frontend/frontend.ml (hunk 1 above)
Comment (OriginLabs-Iguernlala) [reply src/lib/frontend/frontend.ml:228]: Because I'll replace this with another test to print interpretations

**Pull request merged**